src/components/kytos/misc/StatusMenu.vue

```diff
@@ -0,0 +1,764 @@
+                      <th>
+                        <input v-model="interfaceTextFilter[1][1]"></input>
+                      </th>
+                      <th>
+                        <input v-model="interfaceTextFilter[2][1]"></input>
+                      </th>
+                      <th>
+                        <input v-model="interfaceTextFilter[3][1]"></input>
+                      </th>
+                      <th>
+                        <input v-model="interfaceTextFilter[4][1]"></input>
+                      </th>
+                      <th>
+                        <input v-model="interfaceTextFilter[5][1]"></input>
+                      </th>
+                    </tr>
+                  </thead> 
+                  <tbody>
+                    <tr v-for="(item, index) in filtered_interfaceData" :key="item.id">
+                      <td><k-button class="statusTableButton" :icon="isStoredI(item, index)" v-on:click.native="add_remove_Interfaces(item, index)"></k-button></td> 
+                      <td v-on:click="show_infoPanel(item.switch)">{{item.switch}}</td>
+                      <td>{{item.port}}</td>
+                      <td :class="statusColor(item)">{{item.status}}</td>
+                      <td>{{item.status_reason | splitStatusReasons}}</td>
+                      <td>{{item.enabled}}</td>
+                      <td>{{item.active}}</td>           
+                    </tr>
+                  </tbody>           
+              </table>
+            </div>
+          </k-accordion-item>
+        </k-accordion-item>
+      </k-accordion>
+    </div>
+  </div>
+</k-toolbar-item>
+</template>
+
+<script>
+import KytosBase from '../base/KytosBase'
+import KytosBaseWithIcon from '../base/KytosBaseWithIcon'
+
+/**
+ * Menu with a collection of tables displaying the network devices and their status. The menu can be shown or
+ * hidden using the shortcut *Ctrl+Shift+S*.
+ */
+export default {
+  name: 'k-status-menu',
+  mixins: [KytosBaseWithIcon],
+  data () {
+    return {
+    show: false,
+    menuBarInfo: {eventName: 'StatusMenu', icon: 'signal', tooltip: 'Status Menu'},
+    currentSwitchData: "",
+    currentLinkData: "",
+    currentInterfaceData: "",     
+    switches: {},
+    links: {},
+    interfaces: {},
+    switchTextFilter: [
+      ["name", ""],
+      ["status", ""],
+      ["status_reason", ""],
+      ["enabled", ""],
+      ["active", ""]
+    ],
+    linkTextFilter: [
+      ["name", ""],
+      ["status", ""],
+      ["status_reason", ""],
+      ["enabled", ""],
+      ["active", ""]
+    ],
+    interfaceTextFilter: [
+      ["switch", ""],
+      ["port", ""],
+      ["status", ""],
+      ["status_reason", ""],
+      ["enabled", ""],
+      ["active", ""]
+    ],
+    selectedSwitches: {},
+    selectedLinks: {},
+    selectedInterfaces: {},
+    change: 1
+    }
+  },
+  methods: {
+    refresh() {
+      // Refreshes all table data
+      this.get_topology()
+      this.get_interfaces()
```

Comment:
I see you have two API calls to the topology to get the desired information. I wonder what you think about us trying to optimize this and leave the interface processing to be done at the client side. Please see my comment on the get_topology() method definition.

I'm sorry if you guys already discussed this and decided for going this direction.

```suggestion
      this.get_topology()
```